docs/markdown/Creating-releases.md

```diff
@@ -5,27 +5,47 @@ short-description: Creating releases
 # Creating releases
 
 In addition to development, almost all projects provide periodical
-source releases. These are standalone packages (usually either in tar
-or zip format) of the source code. They do not contain any revision
-control metadata, only the source code.
+source releases. These are standalone packages (usually either in
+tar or zip format) of the source code. They do not contain any
+revision control metadata, only the source code.  Meson provides
+a simple way of generating these, with the meson dist command.
```

Comment:
```suggestion
a simple way of generating these, with the `meson dist` command.
```

Reply:
Highlights had been added to the appropriate command.